services/rollout-service/pkg/versions/versions.go

```diff
@@ -21,9 +21,17 @@ import (
 	"fmt"
 
 	"github.com/freiheit-com/kuberpult/pkg/api"
+	"github.com/freiheit-com/kuberpult/pkg/auth"
 	"k8s.io/utils/lru"
 )
 
+// This is a the user that the rollout service uses to query the versions.
+// It is not written to the repository.
+var RolloutServiceUser auth.User = auth.User{
```

Comment:
I think for now this is fine, because the GetOverview service doesn't make any changes in git.
Once we call an endpoint that actually makes a git commit (e.g. batchService), we need to make this name/email configurable like it is in the frontendService (or you call the frontendservice instead).